### DISCUSSION TITLE
Change file icon

### DISCUSSION THREAD

**Discussion by Froc88** [rejetto/hfs#250] [General]
Is there an option to change the icon of the files, such as pdf, jpg, etc.?

**Comment by rejetto** [comment]
at the moment there's no option, but I will soon study it

**Comment by rejetto** [comment]
can you tell me what you expect from such feature exactly?
just nicer and different icons for these files, or do you want to set your custom icons?

**Reply by Froc88** [reply]
It is to see the original icons of the extensions
![image](https://github.com/rejetto/hfs/assets/129078816/d20b548b-a437-430e-b64b-925124fc2af9)

**Comment by rejetto** [comment]
what type of files do you expect to display a dedicated icon?
do you expect to be able to customize the icons yourself?

**Reply by GeorgeGeorgeGe** [reply]
the most common file types like pdf, excel, word, txt, jpg, jpeg, videos.....etc

**Reply by rejetto** [reply]
Maybe you expect the same icons you see in your file explorer.
I don't think I will be able to read those "system" icons.
I made a search once and didn't find anything suitable for HFS 3 (that is for node.js, because that's how HFS3 runs).

**Comment by rejetto** [comment]
what do you think of this?
<img width="803" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/ad39fd47-73de-482a-9fd7-32a113bb4e7f">

**Reply by GeorgeGeorgeGe** [reply]
it is good, but the pdf and MS office files, the most common files, still have that general icon.

**Reply by rejetto** [reply]
i don't mean to introduce icons for single type of files, but categories.
Not like this at least, maybe if I ever get the icons from the operating system.
Anyway, icons are now customizable by plugins. We'll see more.
